packages/cursorless-engine/src/actions/Replace.ts

```diff
@@ -63,33 +63,41 @@ export default class Replace {
     await runForEachEditor(
       edits,
       (edit) => edit.editor,
-      async (editor, edits) => {
-        const contentSelections = {
-          selections: edits.map(({ target }) => target.contentSelection),
-        };
-        const editSelections = {
-          selections: edits.map(({ edit }) => edit.range.toSelection(false)),
-          rangeBehavior: RangeExpansionBehavior.openOpen,
-        };
+      async (editor, editWrappers) => {
+        const edits = editWrappers.map(({ edit }) => edit);
+        const contentSelections = editWrappers.map(
+          ({ target }) => target.contentSelection,
+        );
+        const editRanges = edits.map(({ range }) => range);
+        const editableEditor = ide().getEditableTextEditor(editor);
```

Comment:
just a stylistic nit but i would personally inline this one as well; there is something to be said for making the function a bit smaller as you can fit it on a screen. Always a trade-off tho and very much a stylistic choice so feel free to ignore

my guess is your thinking is that these `performEditsAndUpdateSelections` calls tend to be pretty big so trying to keep the call itself as simple as possible, which makes sense

Reply:
Yeah I was going for readability but I'm happy to inline them